tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
     nose
     nose-exclude
     unittest2
+    -egit+https://github.com/google/oauth2client.git#egg=oauth2client
```

Comment:
Ha sorry. I'm fine just waiting to merge this until an `oauth2client` release gets cut. (We can push to get one soon.)